### ISSUE TITLE
Annoying messages in the console

### ISSUE THREAD

**Issue by sdavidg** [Tampermonkey/tampermonkey#1979]
When a script has permissions like GM getValue, GM getValue,... I can see these messages in the console:

## Actual Behavior
![imagen](https://github.com/Tampermonkey/tampermonkey/assets/4955944/6110c6bc-40c9-4f74-b2a6-373faa68586d)

## Specifications

- Firefox: 122.0
- TM;  5.0.1
- OS:  Win10

## Script

```js
// ==UserScript==
// @name        TEST LOG
// @namespace   sdavidg
// @include     *
// @version     1
// @require     http://code.jquery.com/jquery-3.4.1.min.js
// @require     http://code.jquery.com/ui/1.12.1/jquery-ui.min.js
// @grant GM_getValue
// @grant GM_setValue
// @grant GM_listValues
// @grant GM_setClipboard
// @run-at      document-end
// ==/UserScript==


```

**Comment by nropgrammer** [comment]
Is there an ETA on when this might be fixed? It's incredibly annoying

**Comment by derjanb** [comment]
Fixed at 5.1.6193 ([xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi)) for now.

Please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by sdavidg** [comment]
It's works fine

Thanks
